packages/core/src/hooks/useSendTransaction.ts

```diff
@@ -2,18 +2,27 @@ import { TransactionRequest } from '@ethersproject/abstract-provider'
 import { TransactionOptions } from '../../src'
 import { useEthers } from './useEthers'
 import { usePromiseTransaction } from './usePromiseTransaction'
+import { useConfig } from '../providers/config/context'
 
 /**
  * @public
  */
 export function useSendTransaction(options?: TransactionOptions) {
   const { library, chainId } = useEthers()
   const { promiseTransaction, state, resetState } = usePromiseTransaction(chainId, options)
+  const { percentageGasLimit } = useConfig()
 
   const sendTransaction = async (transactionRequest: TransactionRequest) => {
```

Comment:
The user for some cases might specifically set `gasLimit` to one of the transactions.
In that case, we don't want to estimate the gas limit, even if the percentage config is set.

Reply:
I totally agree